Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```
@@ -527,6 +528,11 @@ public Func<HttpClient> HttpClientFactory
             }
         }
 
+        /// <summary>
+        /// Enable partition key level failover
+        /// </summary>
+        internal bool EnablePartitionLevelFailover { get; set; } = false;
```

What are the semantics in-cases where its not applicable? 
Will it fail the client initialization or silently ignores it?

Is it included in the user-agent?

---

There currently is not any backend flag to know if it should be allowed or not. There shouldn't be any harm in having it enabled for accounts without the backend enabled. The SDK must be enabled if the backend is otherwise the SDK retry logic will break.

It's not currently being included in the user agent string.